README.md

```diff
@@ -17,19 +17,46 @@ The book should build with any recent LaTeX installation. The book is expected t
 * LaTeXML version 0.8.7
 * TeX Live 2022
 
-All documents can be built by running the script `local-build.sh --use-pdf-latex`. This will create a directory called `local-build` that contains the book in PDF, single page HTML, and multi-page HTML formats.
+All documents can be built locally by running the script `local-build.sh --use-pdf-latex`. This will create a directory called `local-build` that contains the book in PDF, single page HTML, and multi-page HTML formats.
 
 The Book also builds in PDF format without using the `local-build.sh` script using: 
 
 * TeXstudio version 4.5.1 or later
 * TeXShop version Version 5.12 or later
 
-The first time the book is built in a clean repository using TeXstudio or TeXShop the glossary build may error out or not be included into the PDF. Just build again and it will fix itself.
+The first time the book is built in a clean repository using TeXstudio or TeXShop the glossary build may error out or not be included into the PDF. Just build again and it should fix itself.
```

Comment:
This can be addressed by using `latexmk` instead of using `pdflatex`.

Reply:
Adjust the text in the next PR to recommend this.